redis-cache.php

```diff
@@ -13,9 +13,11 @@
  * GitHub Plugin URI: https://github.com/rhubarbgroup/redis-cache
  * License: GPLv3
  * License URI: http://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * @package RhubarbGroup/RedisCache
  */
 
-defined( 'ABSPATH' ) || exit;
+defined( 'ABSPATH' ) || exit( 'direct access forbidden' );
```

Comment:
Can we stick to just an exit without a message?

Reply:
Relic of my testing - removed